javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var filterOutStringsResult = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] !== typeof '') {
```

Comment:
The `typeof` operator returns an exact string value, so you can compare `typeof values[i]` to that.

```js
if (typeof values[i] !== 'string') {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof